### ISSUE TITLE
Invalid name of org_name

### ISSUE THREAD

**Issue by microcoder** [jupyterlab/theme-cookiecutter#17]
Hi!

 I input text "microcoder" in a field  `org_name`, after that begin install the package and I got an error:

```
jupyter labextension install .

An error occured.
ValueError: "/home/alpine/juptheme/jupyterlab-ubu-theme" is not a valid npm package
See the log file for details:  /tmp/jupyterlab-debug-4e18d3kv.log
```
 /tmp/jupyterlab-debug-4e18d3kv.log:

```
Node v12.15.0

Yarn configuration loaded.
> node /usr/lib/python3.8/site-packages/jupyterlab/staging/yarn.js install
yarn install v1.21.1
error package.json: Name contains illegal characters
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

> /usr/bin/npm pack /home/alpine/juptheme/jupyterlab-ubu-theme
npm ERR! Invalid name: "microcoder/jupyterlab-ubu-theme"

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/alpine/.npm/_logs/2020-04-03T06_20_28_921Z-debug.log

Traceback (most recent call last):

  File "/usr/lib/python3.8/site-packages/jupyterlab/debuglog.py", line 47, in debug_logging
    yield

  File "/usr/lib/python3.8/site-packages/jupyterlab/labextensions.py", line 93, in start
    ans = self.run_task()

  File "/usr/lib/python3.8/site-packages/jupyterlab/labextensions.py", line 138, in run_task
    return any([

  File "/usr/lib/python3.8/site-packages/jupyterlab/labextensions.py", line 139, in <listcomp>
    install_extension(

  File "/usr/lib/python3.8/site-packages/jupyterlab/commands.py", line 388, in install_extension
    return handler.install_extension(extension, pin=pin)

  File "/usr/lib/python3.8/site-packages/jupyterlab/commands.py", line 603, in install_extension
    info = self._install_extension(extension, tempdir, pin=pin)

  File "/usr/lib/python3.8/site-packages/jupyterlab/commands.py", line 1499, in _install_extension
    info = self._extract_package(extension, tempdir, pin=pin)

  File "/usr/lib/python3.8/site-packages/jupyterlab/commands.py", line 1578, in _extract_package
    raise ValueError(msg % source)

ValueError: "/home/alpine/juptheme/jupyterlab-ubu-theme" is not a valid npm package

Exiting application: jupyter
```

**Comment by maegul** [comment]
Plus one from me.  Same issue.

**Comment by blink1073** [comment]
Thanks for the report!  It looks like we need to add an `@` in this [line](https://github.com/jupyterlab/theme-cookiecutter/blob/e07e12feba4ae6710e9e27bd36126cbbfd5c071f/%7B%7Bcookiecutter.extension_name%7D%7D/package.json#L2).  Would either of you be available to make a PR?

**Comment by blink1073** [comment]
Fixed by @TheLastDestroyer  in #18.
